.github/workflows/integration_tests.yml

```diff
@@ -0,0 +1,77 @@
+on:
+  push:
+    branches-ignore: [main]
```

Comment:
I'm fine with either option. I think the only case, where we wouldn't want to skip, is if we push directly to master. I'm assuming if we don't skip, the tests will run a second time unnecessarily since tests are required to pass in the PR before merging to main.

Reply:
I lean towards not skipping.  I think a little redundancy is ok.  It could catch potential issues related to a dependency updating between when the PR tests last ran and when the PR is merged into main.  I think not skipping would be more important if we didn't require PRs to be up to date with main.

Reply:
Okay, sounds good. Will update to not skip.